tests/functional/cylc-install/00-simple.t

```diff
@@ -148,21 +130,3 @@ INSTALLED $RND_WORKFLOW_NAME/run2 from ${RND_WORKFLOW_SOURCE}
 __OUT__
 popd || exit 1
 purge_rnd_workflow
-
-# -----------------------------------------------------------------------------
-# Test running cylc install twice increments run dirs correctly
-TEST_NAME="${TEST_NAME_BASE}-install-C-twice-1"
-make_rnd_workflow
-run_ok "${TEST_NAME}" cylc install -C "${RND_WORKFLOW_NAME}"
-contains_ok "${TEST_NAME}.stdout" <<__OUT__
-INSTALLED $RND_WORKFLOW_NAME/run1 from ${RND_WORKFLOW_NAME}
-__OUT__
-TEST_NAME="${TEST_NAME_BASE}-install-C-twice-2"
-run_ok "${TEST_NAME}" cylc install -C "${RND_WORKFLOW_NAME}"
-contains_ok "${TEST_NAME}.stdout" <<__OUT__
-INSTALLED $RND_WORKFLOW_NAME/run2 from ${RND_WORKFLOW_NAME}
-__OUT__
-purge_rnd_workflow
-
-
-exit
```

Comment:
Is this exit not needed? Seems like the bulk of our tests (but not all) have it, guessing it is to ensure if error occurs then it exits properly.

Reply:
I'm pretty sure an exit statement with no argument at the end of a script is useless. And annoying.  If no one can think of a good reason for them, let's remove them.